.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: "Clone Repository"
       uses: actions/checkout@v3
     - name: "Run"
-      uses: osbuild/containers/src/actions/privdocker@552e30cf1b4ed19c6ddaa57f96c342b3dff4227b
+      uses: lukewarmtemp/containers/src/actions/privdocker@552e30cf1b4ed19c6ddaa57f96c342b3dff4227b
```

Comment:
Note: Will change this back before making a PR upstream. Hoping to run CI through my fork after all suggested changes are approved.